compute_sdk/tests/unit/test_client.py

```diff
@@ -783,3 +822,11 @@ def test_auth_client_deprecated():
         assert gcc.auth_client, "Client.auth_client needed for backward compatibility"
     msg = "'Client.auth_client' attribute is deprecated"
     assert any(msg in str(r.message) for r in record)
+
+
+def test_login_manager_deprecated():
+    gcc = gc.Client(do_version_check=False)
+    with pytest.warns(DeprecationWarning) as record:
+        gcc.login_manager, "Client.login_manager needed for backward compatibility"
```

Comment:
I don't recognize that syntax ... is that a test-focused comment in the form of a `tuple()`, but the test is the side-effect of the `.login_manager` property?  Or is that missing an `assert` as in the nearby test functions?